src/RevisionPopupWidget.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @class
+ * @constructor
+ */
+const RevisionPopupWidget = function RevisionPopupWidget() {
+	// Parent constructor
+	RevisionPopupWidget.parent.call( this, {
+		padded: true,
+		autoClose: true,
+		position: 'above',
+		// FIXME: 'force-left' for RTL languages
+		align: 'force-right',
+		hideWhenOutOfView: false,
+		$content: $( '<div>' )
+			.addClass( 'ext-wwt-revisionPopupWidget-content' )
+	} );
+};
+
+/* Setup */
+OO.inheritClass( RevisionPopupWidget, OO.ui.PopupWidget );
+
+/**
+ * Update the contents of the revision popup.
+ * @param {Object} data As returned by Api.getTokenInfo().
+ */
+RevisionPopupWidget.prototype.setContent = function ( data ) {
+	// Note that the English namespaces will normalize to the wiki's local namespaces.
+	const userLinks = `
+		<a target="_blank" href="${mw.util.getUrl( `User:${data.username}` )}">${data.username}</a>
+		${mw.msg( 'parentheses-start' )}<a target="_blank" href="${mw.util.getUrl( `User talk:${data.username}` )}">${mw.msg( 'talkpagelinktext' )}</a>
+		${mw.msg( 'pipe-separator' )}
+		<a target="_blank" href="${mw.util.getUrl( `Special:Contributions/${data.username}` )}">${mw.msg( 'contribslink' )}</a>${mw.msg( 'parentheses-end' )}
+	`,
+		// eslint-disable-next-line no-undef
```

Comment:
Is it complaining about "moment" ?  If so, add it to `.eslintrc` as `"moment": false` in globals